packages/sanity/src/core/form/studio/assetSourceDataset/uploader.ts

```diff
@@ -22,7 +22,7 @@ export class DatasetUploader implements AssetSourceUploader {
   private client: SanityClient
   private subscriptions: Map<string, Subscription> = new Map()
 
-  public constructor(private props: CreateDatasetAssetSourceProps) {
```

Comment:
this.props aren't used within the `DatasetUploader` itself, and since it's private it's not accessible externally either, should be ok to remove